plugins/pmsmart/src/test/java/org/itsallcode/whiterabbit/plugin/pmsmart/integration/JsonFileStorageMock.java

```diff
@@ -0,0 +1,36 @@
+package org.itsallcode.whiterabbit.plugin.pmsmart.integration;
+
+import java.nio.file.Path;
+import java.time.YearMonth;
+import java.util.Optional;
+
+import javax.json.bind.Jsonb;
+
+import org.itsallcode.whiterabbit.api.model.MonthData;
+import org.itsallcode.whiterabbit.logic.storage.data.JsonFileStorage;
+import org.itsallcode.whiterabbit.logic.storage.data.JsonModelFactory;
+import org.itsallcode.whiterabbit.logic.storage.data.JsonbFactory;
+
+public class JsonFileStorageMock extends JsonFileStorage
```

Comment:
Same here: use Mockito to create a mock.

Reply:
Replaced by local static final class and narrow interfaces.